examples/cloudflare-edge/README.md

```diff
@@ -0,0 +1,50 @@
+# Hydrogen v2 on Cloudflare edge
+
+- [Hydrogen Docs](https://shopify.dev/docs/custom-storefronts/hydrogen)
+- [Remix Docs](https://remix.run/docs)
+
+## Install
+
+```sh
+npm install --legacy-peer-deps
```

Comment:
`@shopify/hydrogen` has specific peer dependency with `"@remix-run/react": "1.12.0"` and for cloudflare edge I'm using `^1.13.0`. This command can be updated as soon as hydrogen gets updated.